BardMusicPlayer.Seer/Lib/Machina/TCPNetworkMonitor.cs

```diff
@@ -116,36 +116,20 @@ public void Stop()
         _connectionManager.Cleanup();
     }
 
-    private void ProcessDataLoop(CancellationToken token)
+    private async Task ProcessDataLoop(CancellationToken token)
     {
         try
         {
             while (!token.IsCancellationRequested)
             {
-                try
-                {
-                    _connectionManager.Refresh();
-
-                    ProcessNetworkData();
-
-                }
-                catch (OperationCanceledException)
-                {
-
-                }
-                catch (Exception ex)
-                {
-                    if (DateTime.UtcNow.Subtract(_lastLoopError).TotalSeconds > 5)
-                        Trace.WriteLine("TCPNetworkMonitor Error in ProcessDataLoop inner code: " + ex.ToString(), "DEBUG-MACHINA");
-                    _lastLoopError = DateTime.UtcNow;
-                }
-
-                Task.Delay(30, token).Wait(token);
+                _connectionManager.Refresh();
+                ProcessNetworkData();
+                await Task.Delay(30, token);
             }
         }
         catch (OperationCanceledException)
         {
-
+            // Handle cancellation
```

Comment:
this is intentionally blank, this meaning the caller of this async processes wants this task to stop immediately.

Reply:
Rider suggested the declaration can be moved closer to usage, which moves it right above the two while loop checks inside the main loop. The inside loops use the tcpbuffer.

Reply:
Removed comment.